micro-rdk/src/common/webrtc/grpc.rs

```diff
@@ -21,8 +21,8 @@ use crate::{
 use super::{api::WebRtcError, sctp::Channel};
 
 #[cfg(feature = "camera")]
-// sizeof(fake_image) + 30bytes for headers
-static WEBRTC_GRPC_BUFFER_SIZE: usize = 10492;
+// sizeof(fake_image) + headers/encodings
+static WEBRTC_GRPC_BUFFER_SIZE: usize = 1024 * 11;
```

Comment:
running continuously on RC page revealed an occasional size of 10493 for `render_frame`. 
Bumped to the next largest multiple of 1024.

Reply:
do we know why?

Reply:
narrowing it down. something to do with the final encoded length.
I've verified that the sizeof `HttpBody` message is constant at `10477`, but the final encoded message length goes from `10492` to `10493` and continues at the latter from that point on (ran for 5min).

Reply:
should I make a ticket to investigate and move forward?

Reply:
nope it's fine